### ISSUE TITLE
Recommended installation type

### ISSUE THREAD

**Issue by radiorambo** [zellij-org/zellij#1704]
There are three ways to install zellij in documentation. Cargo, Binary and third party repositories.

Have a recommended installation type like mentioning in a bracket in the heading, would help newbies to make decision effortlessly.

**Comment by imsnif** [comment]
Hey, @RadioRambo - umm... I've been thinking about this ever since I saw your issue pop-up yesterday and I'll admit I don't have a very good answer!

This is one of those "it depends" things. If you have a package manager installed (eg. brew, pacman) you'd probably be most comfortable installing from there. If you have cargo installed, you'd likely be most comfortable installing from there. I you trust us with binary files, a quick way you be to use the binstall or to download the precompiled executable directly. I'm just not sure there is a catch-all here, you know?

**Comment by RadioRambo** [comment]
Hey,

I agree with your view. There is no one fit for all. So I would say guiding them like adding a paragraph at the beginning of the page would be helpful.

> _Rough suggestion. Can be improved_

> There are total four ways to install zellij. 
> 1. If you have already installed rust, use method 1. 
> 2. If you have some time and enough computer power, use method 2 and compile yourself. 
> 3. Else download binary we prebuilt, use method 3. 
> 4. Or if your prefer third party repositories and if it is listed, use method 4.

> If you are confused, We would suggest method #, as it is simplest way to install.

**Comment by imsnif** [comment]
Hey @RadioRambo - I'm honestly a little afraid this will clutter up the README. Do you have examples of other projects doing something like this?

**Comment by RadioRambo** [comment]
No. But this is what bugs me. And that is why I feel majority of the Linux blogs are of "How to install xxxxx" and those blogs are generating a lot of view for a reason that I feel every developer has taken for granted. When you look the installation page of most Linux software, you will get confused in perspective of a new Linux user. Any new user will smoothly will navigate to official installation page of any given software and get struck on that page. These architectures, binaries, packages mangers and repositories are often confusing. . And also one of the quality that makes anyone as advanced user in Linux world is being able to install software without confusing, which oddly is considered as common task in other operating systems. flatpak and snapstore is trying to solve this issue but that isn't happening any time soon. 

I understand you and your reason but that extra clutter you feel is user friendly to new users. There are ways to deal with extra text like accordions, tabs, tooltips if that is what holding you back.

I feel that the reason that people find hard in Linux isn't installing the operating system but installing required applications.

Don't get me wrong. And thanks for consideration.
